source/features/inline-instagram-photos.js

```diff
@@ -24,17 +24,23 @@ export default function () {
 	$('a.twitter-timeline-link[data-expanded-url*="//www.instagram.com').each(async (idx, instagramAnchor) => {
 		const tweetElement = $(instagramAnchor).parents('.js-tweet-text-container');
 		const instagramPostUrl = instagramAnchor.dataset.expandedUrl;
-		const hasMediaSibling = tweetElement.siblings('.AdaptiveMediaOuterContainer').length > 0;
+		const shouldInlinePhoto = tweetElement.attr('data-refined-twitter-inlined-instagram') === undefined;
+		if (shouldInlinePhoto) {
+			tweetElement.attr('data-refined-twitter-inlined-instagram', 'waiting');
```

Comment:
I've used a "two stage" custom data attribute to handle the racing condition, I'm just not a huge fan of this long name. Any suggestions?